src/autodiscovery.js

```diff
@@ -428,13 +429,22 @@ export class AutoDiscovery {
      * be an empty object.
      */
     static async getRawClientConfig(domain) {
+        const default_server_config = SdkConfig.get()['default_server_config'];
+        const base_url_from_config = default_server_config['m.homeserver']['base_url'] || SdkConfig.get()['default_server_name'];
+        const server_name = base_url_from_config.replace('https://','');
+        console.log("config value for base_url:::", base_url_from_config);
+
         if (!domain || typeof(domain) !== "string" || domain.length === 0) {
             throw new Error("'domain' must be a string of non-zero length");
         }
 
+        if(!server_name.includes(domain)){
```

Comment:
I don't understand why this if statement is required.

Reply:
if chs.local.gov.au is part of default_server then just use that otherwise switch the value.